t/test_helper.rb

```diff
@@ -2,6 +2,7 @@
 
 ENV['RACK_ENV'] = 'test'
 
+require 'minitest/around'
```

Comment:
The tests are no longer running cleanly - [we're getting a warning](https://travis-ci.org/everypolitician/legislative-explorer/builds/338388197#L681) and I think it's related to the load order.

I think this needs to be `require`d _after_ `minitest/autorun` so that minitest is available for `minitest/around` to use.

Reply:
Yick. I'm so used to seeing that warning on the `viewer-sinatra` tests that my brain didn't even register that it was newly introduced here. One more reason why it's good to never let test suites get to that state! I've switched these around in a fixup commit.